packages/icrc-ledger-types/src/icrc1/account.rs

```diff
@@ -167,14 +170,66 @@ impl FromStr for Account {
     }
 }
 
+const MAX_SERIALIZATION_LEN: u32 = 62;
+
+impl Storable for Account {
+    fn to_bytes(&self) -> Cow<[u8]> {
+        let mut buffer: Vec<u8> = vec![];
+        let mut buffer0: Vec<u8> = vec![];
+
+        if let Some(subaccount) = self.subaccount {
+            buffer0.extend(subaccount.as_slice());
+        }
+        buffer0.extend(self.owner.as_slice());
+        buffer.extend((buffer0.len() as u8).to_le_bytes());
```

Comment:
I was wondering if it would make sense to add a check to make sure that `buffer0.len()` fits into a `u8`, but I then saw that `ic_principal::Principal::MAX_LENGTH_IN_BYTES` is `29`, and the `Subaccount` length is `32`, and neither will probably change anytime soon, and the fact that `Account` is `Bounded` with a `max_size` of `62`, so if the length goes beyond that we should already get an error, so probably a check here is not necessary.